config.js

```diff
@@ -369,6 +373,12 @@ var config = {
 
     // }
 
+    // dropbox: {
+    //     // Replace <APP_ID> with your dropbox application ID to enable the
+    //     // dropbox integration.
+    //     clientId: "<APP_ID>"
+    // }
+
```

Comment:
@hristoterezov, what?! Why are there 2  effectively the same (minus the comments that is) here and above? I must be missing something by reading the GitHub diff.

Reply:
You are not! My mistake...